app/mesh/changelog.md

```diff
@@ -4,6 +4,20 @@ no_search: true
 no_version: true
 ---
 
+## 1.2.4
+
+> Released on 2021/04/19
+
+### Changes
+
+Built on top of [Kuma 1.1.4](https://github.com/kumahq/kuma/blob/master/CHANGELOG.md#112).
+
+Includes important bug fixes to version 1.1.3 of Kuma, plus improvements to the web UI.
```

Comment:
Is this important bug fixes to v 1.1.4 of Kuma?

Reply:
oops, THANK YOU